test/stat_tracker_test.rb

```diff
@@ -48,4 +50,18 @@ def test_it_can_show_average_goals_by_season
     expected = {20122013=>4.71, 20132014=>4.00}
     assert_equal expected, @stat_tracker.average_goals_by_season
   end
+
+  def test_it_can_get_home_wins_percentage
+    assert_equal 0.6, @stat_tracker.percentage_home_wins
+  end
+
+  def test_it_can_get_away_win_percentages
+    assert_equal 0.3, @stat_tracker.percentage_visitor_wins
+  end
+
+  def test_it_can_get_tied_percentage
+    @stat_tracker.stubs(:percentage_ties).returns(0.10)
+
+    assert 0.10, @stat_tracker.percentage_ties
```

Comment:
we can't use a stub here unfortunately. We are only testing that a stub returns the value that we want and not actually testing the method at all. we should all discuss modifying the truncated csv files. I am also running into some inadequacies in the data pool we are pulling from.